packages/bento-design-system/src/react-input-mask.d.ts

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "react-input-mask";
+
+declare module "react-input-mask" {
+  interface Props {
+    children: (inputProps: InputHTMLAttributes<HTMLInputElement>) => JSX.Element;
+  }
+}
```

Comment:
needed since React 18 removed the "implicit" `children` prop, and `react-input-mask` types have not been updated in a while.